src/ring_theory/ideal/operations.lean

```diff
@@ -859,6 +859,27 @@ end
 
 end ideal
 
+lemma ideal_mul_eq_bot {R : Type*} [integral_domain R] {I J : ideal R} : (I * J = ⊥) ↔ I = ⊥ ∨ J = ⊥ :=
```

Comment:
This can go after `pow_le_pow`, line 340. Then we can drop the `ideal` from the name:
```suggestion
lemma mul_eq_bot {R : Type*} [integral_domain R] {I J : ideal R} : (I * J = ⊥) ↔ I = J ∨ J = ⊥ :=
```